### PR TITLE
Doc: Updating the landing page, move the collection below AVD umbrella

### PR DESCRIPTION
**Pull request by JulioPDX**
## Change Summary

Updating landing page to move the Ansible collection below the AVD umbrella.

## Proposed changes

- Update layout to clarify The Ansible collection vs AVD as a framework

### Repository Checklist

- [x] My code has been rebased from devel before I start
- [x] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [x] My change requires a change to the documentation and documentation have been updated accordingly.
- [x] I have updated [molecule CI](https://github.com/aristanetworks/ansible-avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)

### CODE REVIEW

**Comment by JulioPDX** [comment]
@carlbuchmann Work in progress to move the ansible collection below "AVD." Let me know what you think, I will also update the GitHub readme after changes are approved.

File: mkdocs.yml
Comment (ClausHolbechArista) [inline mkdocs.yml]: Still says collection

File: ansible_collections/arista/avd/README.md
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/README.md]:
```suggestion
- [development version](https://www.avd.sh/en/devel/)
```

File: ansible_collections/arista/avd/README.md
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/README.md:66]: Should we mention that if you have a support agreement you can contact Arista support?

File: ansible_collections/arista/avd/README.md
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/README.md]:
```suggestion
Arista Validated Designs (AVD) is an extensible data model that defines Arista’s Unified Cloud Network architecture as "code".
```

File: ansible_collections/arista/avd/README.md
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/README.md]: This is still part of the collection
```suggestion
### Custom plugins & modules
```

File: ansible_collections/arista/avd/README.md
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/README.md]:
```suggestion
Arista Validated Designs (AVD) is an extensible data model that defines Arista's Unified Cloud Network architecture as "code".
```

**Comment by ClausHolbechArista** [comment]
@JulioPDX

File: ansible_collections/arista/avd/README.md
Comment (JulioPDX) [reply ansible_collections/arista/avd/README.md:66]: @ClausHolbechArista Just pushed a small note on this topic.  Please let me know if it requires any edits

File: mkdocs.yml
Comment (ClausHolbechArista) [inline mkdocs.yml]:
```suggestion
site_description: Arista Validated Designs documentation
```

Comment (ClausHolbechArista) [reply mkdocs.yml]: You resolved this one with no update. I have comitted a change to line 4 to remove the "collection's" and add "s" on "Designs". I will approve now, but let me know if there was a reason why you didn't change the description.

**Review by ClausHolbechArista — APPROVED** [review]

**Review by nielsjlarsen — APPROVED** [review]
LGTM!

**Pull request merged**
